etc/kayobe/kolla.yml

```diff
@@ -258,6 +262,11 @@ kolla_build_blocks:
     RUN sed -i 's/"pc-q35-rhel8.5.0"/"pc-q35-*"/' /usr/share/qemu/firmware/50-edk2-ovmf-cc.json
     {% endif %}
     {% endraw %}
+  horizon_header: |
+    # Fix for https://bugs.launchpad.net/horizon/+bug/1997545
```

Comment:
worth a note here that the version comes from Zed u-c

Reply:
Done